docs/datastorage/extendedentityproperties.md

```diff
@@ -0,0 +1,214 @@
+
+1. When the entity spawns in the world, you may want to share the initialization-assigned values with the clients.
+1. When the stored data changes, you may want to notify some or all of the watching clients.
+1. When a new client starts viewing the entity, you may want to notify it of the existing data.
+
+Refer to the [Networking](../networking/index.md) page for more information on implementing the network packets.
+
+For example:
+
+```Java
+private void dataChanged()
+{
+  if(!world.isRemote)
+  {
+    EntityTracker tracker = ((WorldServer)world).getEntityTracker();
+    ExampleEntityPropertySync message = new ExampleEntityPropertySync(this);
+
+    for (EntityPlayer entityPlayer : tracker.getTrackingPlayers(entity))
+    {
+      ExampleMod.channel.sendTo(message, (EntityPlayerMP)entityPlayer);
+    }
+  }
+}
+
+private void entitySpawned()
+{
+  dataChanged();
+}
+
+private void playerStartedTracking(EntityPlayer entityPlayer)
+{
+  ExampleMod.channel.sendTo(new ExampleEntityPropertySync(this), (EntityPlayerMP)entityPlayer);
+}
+```
+
+And the corresponding event handlers:
+
+```Java
+@SubscribeEvent
+public void entityJoinWorld(EntityJoinWorldEvent e)
+{
+  ExampleEntityProperty data = ExampleEntityProperty.get(e.entity);
+  if (data != null)
+    data.entitySpawned();
+}
+
+@SubscribeEvent
+public void playerStartedTracking(PlayerEvent.StartTracking e)
+{
+  ExampleEntityProperty data = ExampleEntityProperty.get(e.target);
+  if (data != null)
+    data.playerStartedTracking(e.entityPlayer);
+}
+```
+
+Persisting across Player Deaths
+-------------------------------
+
+By default, the entity data does not persist on death. In order to change this, the data has to be manually copied when the player entity is cloned during the respawn process.
+
+This can be done by handling the `PlayerEvent.Clone` event. In this event, the `wasDead` field can be used to distinguish between respawning after death, and returning from the End. This is important because the data will already exist, so care has to be taken to not duplicate values when returning from the End dimension.
+
+```Java
+@SubscribeEvent
+public void onClonePlayer(PlayerEvent.Clone e)
+{
+  if(e.wasDeath)
+  {
+    NBTTagCompound compound = new NBTTagCompound();
+    ExampleEntityProperty.get(e.original).saveNBTData(compound);
+    ExampleEntityProperty.get(e.entityPlayer).loadNBTData(compound);
+  }
+}
+```
+
+Updating the Contents Over Time
+-------------------------------
+
+By default, the entity data does not tick. In order to change this, the data has to be manually made to update.
+
+This can be done by handling the `TickEvent.PlayerTickEvent` event if the entity is attached to players.
+
+```Java
+private void update()
+{
+  setStoredData(Integer.toString(entity.ticksExisted));
+}
+```
+
+```Java
+@SubscribeEvent
+public void playerTickEvent(LivingEvent.LivingUpdateEvent e)
```

Comment:
You talk about the PlayerTickEvent but then use LivingUpdateEvent. It's not necessary to explain what these events are but you should at least be consistent.